handlers/directMessageHandler.js

```diff
@@ -6,10 +6,11 @@
 
 /**
  * Handles direct messages sent to Bastion
+ * @param {methods} any methods globally available
  * @param {Message} message Discord.js message object
  * @returns {void}
  */
-module.exports = message => {
+module.exports = (methods, message) => {
```

Comment:
One PR should address only one thing at a time. Please don't modify the direct message handler.